crates/biome_grit_patterns/tests/specs/ts/spreadMetavariable.ts

```diff
@@ -0,0 +1,11 @@
+const arr1 = [0, 1, 2];
+const arr2 = [1];
+const arr3 = ["a", 1, "b"];
+
+const obj1 = { key: 1, a: 2 };
+const obj2 = { key: 1 };
+const obj3 = { key: 1, x: true, y: false };
+
+fn(1, 2, 3);
+fn();
+fn("hello", "world");
```

Comment:
It may be good to add some cases that _don't_ match as well. Right now the test doesn't confirm whether the objects are matching because there is a `key: 1` in there, or if the query might match any object.